src/config.ts

```diff
@@ -26,15 +26,11 @@ export type HostComponentNames = {
 };
 
 export type InternalConfig = Config & {
-  /** Whether to use breaking changes intended for next major version release. */
-  useBreakingChanges: boolean;
```

Comment:
Don't we want to keep this flag for the next time we introduce breaking changes?

Reply:
I was thinking about pros and cons, and it seems like YAGNI approach makes more sense.

It was created in a particilar project situation in mind. Future cases might be simillar or might be better addressed e.g. by version branches.

We can re-introduce it fairly easily if needed.